lib/task_helpers/imports/alerts.rb

```diff
@@ -0,0 +1,29 @@
+module TaskHelpers
+  class Imports
+    class Alerts
+      class ParsedNonDialogYamlError < StandardError; end
```

Comment:
This class is defined in all of the rhconsulting rake scripts, but only used in one of them. I'll remove this class as it is not used.